From: Calculinux <noreply@calculinux>
Subject: dt: mark Luckfox Lyra OTG VBUS as always on

OTG VBUS is diode-ORed to VCC5V0_SYS. Carry picocalc-drivers#29 on the
currently pinned tree so we do not also pull the M0 rproc build.

---
 linux-rk3506-luckfox-lyra.dtsi | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/linux-rk3506-luckfox-lyra.dtsi b/linux-rk3506-luckfox-lyra.dtsi
index a949261..8e21301 100644
--- a/linux-rk3506-luckfox-lyra.dtsi
+++ b/linux-rk3506-luckfox-lyra.dtsi
@@ -182,6 +182,8 @@
 /**********usb**********/
 &usb20_otg0 {
 	dr_mode = "otg";
+	/* OTG VBUS is diode-ORed to VCC5V0_SYS / VSYS; it never drops. */
+	snps,vbus-always-on;
 	/* vbus-supply = <&vcc5v0_otg0>;
 	rockchip,gpio-vbus-det; */
 	status = "okay";
@@ -198,6 +200,7 @@
 };
 
 &u2phy_otg0 {
+	rockchip,vbus-always-on;
 	status = "okay";
 };
 
--
